shared/util/expo-image-picker.tsx

```diff
@@ -0,0 +1,44 @@
+import {isIOS} from '../constants/platform'
+import * as ImagePicker from 'expo-image-picker'
+import * as Permissions from 'expo-permissions'
+
+export const parseUri = (result: {uri: string}): string => {
+  return isIOS ? result.uri.replace('file://', '') : result.uri.replace('file:', '')
+}
+
+const retyAfterAskingPerm = (retryFn: null | (() => Promise<ImagePicker.ImagePickerResult>)) => (
```

Comment:
This keeps the old functionality where we try to ask for permissions if we didn't have it already.